src/lib/structures/Command.ts

```diff
@@ -131,7 +131,7 @@ export abstract class Command<T = Args> extends AliasPiece {
 	 */
 	protected parseConstructorPreConditionsRunIn(options: CommandOptions) {
 		const runIn = this.resolveConstructorPreConditionsRunType(options.runIn);
-		if (runIn !== null) this.preconditions.append(runIn);
+		if (runIn !== null) this.preconditions.append(runIn as any);
```

Comment:
why is this cast now needed?

Reply:
Because `preconditions.append` has two separate overloads, and `runIn` returns both simultaneously.